Apps/Src/BlinkLights.c

```diff
@@ -11,10 +11,10 @@
 void Task_BlinkLight(void* p_arg){
     OS_ERR err;
     while(1){
-        Lights_Set(LEFT_BLINK, (State) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
-        Lights_Set(RIGHT_BLINK, (State) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
-        Display_SetLight(LEFT_BLINK, (State) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
-        Display_SetLight(RIGHT_BLINK, (State) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
+        Lights_Set(LEFT_BLINK, (bool) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
+        Lights_Set(RIGHT_BLINK, (bool) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
+        Display_SetLight(LEFT_BLINK, (bool) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
+        Display_SetLight(RIGHT_BLINK, (bool) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
```

Comment:
Unnecessary cast to bool (both Lights_Toggle_Read and Lights_Read return a bool)